README.md

```diff
@@ -124,32 +127,64 @@ https://travis-ci.org/profile
 <a name="create-the-project-files"></a>
 ### Create The Project Files
 
-> Now back in your text editor create a few *new files*:
+In this example, the project structure and files you will need are as follow:
 
-```sh
-vi .travis.yml
 ```
-Paste the following code:
+project_folder
+|_.travis.yml
+|_hello.js
+|_package.json
+|_other_files
+```
+
+Once you hooked your project to Travis, every time you push a new version to GitHub
+Travis will search your entire project folder for files, build your project and
+run all tests automatically. This way you don't need to specify which folders
+Travis needs to check - it always **checks everything**!
+
+First, let's create our `.travis.yml` file and paste the following code:
 
 ```yml
 language: node_js
 node_js:
-  - 6
+ - "node"
 ```
 
 **.travis.yml** is a basic Travis configuration file
-that tells travis-ci our application runs on **node.js**  
-and we want them to test it using a *specific version* of node.  
-(*the file needs to be in the* ***root*** *of your GitHub repository*)
+that tells travis-ci what to expect and how to behave for our application.
+
+In this case, this file tells Travis we run on *Node.js*. Not only that, it also
+tells Travis to use the latest version of Node.js. You can also specify specific
+versions if you want to and to customize your build process:
+
+ - https://docs.travis-ci.com/user/languages/javascript-with-nodejs/
+ - https://docs.travis-ci.com/user/customizing-the-build/
+
+Because this specific file is so vital, it is mandatory that you place it in the
+root of your project folder and that you also validate it, either via the
+[Travis-CLI ](#install-travis-cli-on-ubuntu) or via their [WebLint](http://lint.travis-ci.org/).
+
+Second, lets create our `hello.js` file by pasting the following code:
+
+```javascript
+var http = require('http');
```

Comment:
while _we_ may think this "core node.js http" code is simple enough, it _might_ not be "beginner friendly" _enough_... _however_ my "_vote_" is to _keep_ it until a _complete_ beginner tells us they are "_confused_" by it. 👍